src/wymeditor/editor/base.js

```diff
@@ -2032,18 +2032,15 @@ WYMeditor.editor.prototype.paste = function (str) {
 WYMeditor.editor.prototype.insert = function (html) {
     // Do we have a selection?
     var wym = this,
-        selection = wym.selection(),
+        selection = wym.hasSelection() ? wym.selection() : false,
         range,
         node;
-    if (selection.focusNode !== null) {
+    if (selection) {
         // Overwrite selection with provided html
         range = selection.getRangeAt(0);
         node = range.createContextualFragment(html);
         range.deleteContents();
         range.insertNode(node);
-    } else {
-        // Fall back to the internal paste function if there's no selection
-        wym.paste(html);
```

Comment:
This caused the IE bug. IE, when there was no selection, would `paste`.

`paste`, in turn, when there's no selection, inserts at the top of the document.

Not sure why this fallback was [ever made](https://github.com/wymeditor/wymeditor/commit/f86ac09a486b9c0c330e6ef16ed493207c5f2825).

I'm willing to part with it.